README.md

```diff
@@ -44,6 +44,37 @@ postmanparser also validates for the required fields mentioned by postman schema
 ### Reading the data
 Postman collection contains group of requests and one or more folders having group of requests and/or nested folders in it.
 
+#### Getting all the requests in the collection
+
+You can retreive all the requests present in the collection using `get_requests()` method. It will recursively search through all the folders present in the collection and will return list of requests.
+
+```python
+collection = Collection()
+collection.parse_from_file("path/to/postman/schema.json")
+requests = collection.get_requests()
+for request in requests:
+        print(request) #Either a Request object or str
+```
+
+#### Getting requests by folder in the collection
+
+You can retrieve the requests inside specific folder using dict returned by `get_requests_map()` method. The key of the dict is path to the folder with trail backlash and value is list of requests of type `Request` or `str`.
```

Comment:
This is confusing, do you give all requests item of that path and even the path nested inside? for example if I do `requests_map["/folder1/"]` will it also give me requests inside `"folder1/folder2"` also ?

Reply:
Ok figured out, `requests_map()` will never do a recursive maps. Please check my comments below 👍

Reply:
Yes. It's not recursive maps. Dict Keys are basically folder paths to access any folder at any level in one lookup.